src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -268,28 +316,43 @@ public void IsCompatibilitySwitchSet()
         [Fact]
         public void IsDefaultAppDomain()
         {
-            Assert.True(AppDomain.CurrentDomain.IsDefaultAppDomain());
+            RemoteInvoke(() =>
+            {
+                Assert.True(AppDomain.CurrentDomain.IsDefaultAppDomain());
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void IsFinalizingForUnload()
         {
-            Assert.False(AppDomain.CurrentDomain.IsFinalizingForUnload());
+            RemoteInvoke(() => {
+                Assert.False(AppDomain.CurrentDomain.IsFinalizingForUnload());
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void toString()
         {
             string actual = AppDomain.CurrentDomain.ToString();
+
+            // NetFx has additional line endings
+            if (PlatformDetection.IsFullFramework)
```

Comment:
Can you better state what is different in netfx pls. Line Endings like \r\n or \n wouldn't be removed by trim so I guess you mean whitespaces?